docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,256 @@
+
+> If you are running the tutorial from a cluster where HTCondor is not available you can also submit the jobs to the slurm system, just change the `--job-mode condor` to `--job-mode slurm`. 
+
+After all jobs are completed we can combine the files for each POI: 
+
+```shell
+for p in r_zh_75_150 r_zh_150_250noj r_zh_150_250wj r_zh_250_400 r_zh_gt400
+do
+    hadd -k -f scan_${p}_blinded.root higgsCombine.scans_blinded.${p}.POINTS.*.MultiDimFit.mH120.root
+done
+```
+And finally plot the likelihood scans 
+
+```shell
+python scripts/plot1DScan.py scan_r_zh_75_150_blinded.root  -o scan_r_zh_75_150_blinded --POI r_zh_75_150 --json summary_zh_stxs_blinded.json
+```
+![](figures/scan_plot_r_zh_75_150_blinded.png)
+
+## Impacts
+
+One of the important tests before we move to the unblinding stage is to check the impacts of nuisance parameters on each POI. For this we can run the `combineTool.py` with `-M Impacts` method. We start with the initial fit, which should take ~20 minutes (good time to have a coffee break!):
+ 
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --robustFit 1 --doInitialFit --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --X-rtd FAST_VERTICAL_MORPH
+```
+
+> Note that it is important to add the option `--redefineSignalPOIs [list of parameters]`, to produce the impacts for all POIs we defined when the workspace was created with the `multiSignalModel`.
+
+After the initial fit is completed we can perform the likelihood scans for each nuisance parameter as shown below
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --robustFit 1 --doFits --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --job-mode condor --task-name impacts_zh --X-rtd FAST_VERTICAL_MORPH 
+```
+We will submit the jobs to the HTCondor to speed up the process.
+
+Now we can combine the results into the `.json` format and use it to produce the impact plots.
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --output impacts.json 
+
+plotImpacts.py -i impacts.json -o impacts_r_zh_75_150 --POI r_zh_75_150
+```
+
+![](figures/impacts_zh_75_150.png)
+* Do you observe differences in impacts plots for different POIs, do these differences make sense to you? 
+
+## Unfolded measurements
+
+Now that we studied the NP impacts for each POI, we can finally extract the measurements.
+Note, that in this exercise we are skipping further checks and validation that you should do on your analysis for the purposes of the tutorial. Namely the goodness of fit test and the post-fit plots of folded observables. Both of these checks were detailed in the previous exercises, you can find the description under the following links. 
```

Comment:
Sorry, it was my suggestion earlier but maybe its better to remove "for the puproses of the tutorial" here. To make sure no one thinks we are saying that they should do the checks for the purposes of the tutorial. Alternatively we could write it as "Note, that in this exercise, for the purposes of the tutorial, we are skipping ... "

(and I think we are missing the links referred to by "the following links")